ibridges/search.py

```diff
@@ -0,0 +1,54 @@
+""" Data query
+"""
+from typing import Optional
+
+from ibridges.irodsconnector import keywords as kw
+from ibridges import Session
```

Comment:
In the package itself, you generally want to use the full path (ibridges.irodsconnector.session). This might prevent some circular dependencies at some point.